schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  date_of_birth DATE NOT NULL,
+  escape_attempts INT NOT NULL,
+  neutered BOOLEAN NOT NULL,
+  weight_kg DECIMAL(10,2) NOT NULL,
+  species VARCHAR(255) NOT NULL
```

Comment:
- [ ] @DevKaranJ Great work adding column species in your animal's table, however, you did it in the wrong way, we are supposed to use the ` ALTER` keyword, this is in line with best practises on how to add a new column to the database using SQL. eg `   ALTER TABLE animals ADD species varchar(255);`